src/components/uncorrelatedEvents.js

```diff
@@ -18,7 +18,7 @@ class UncorrelatedEvents extends Component {
 
   componentDidMount () {
     const { filters, dispatch } = this.props
-    const newFilter = Object.assign({}, filters, {incidentId: null})
+    const newFilter = Object.assign({}, filters, { incidentId: null })
     dispatch(filterActions.clearFilterIncidentId(newFilter))
```

Comment:
Shouldn't need the line with

const newFilter

and shouldn't need to pass anything into clearFilterIncidentId